v5/pivotal/stories.go

```diff
@@ -125,6 +127,40 @@ func (service *StoryService) List(projectId int, filter string) ([]*Story, *http
 	return stories, resp, err
 }
 
+type StoryCursor struct {
+	*cursor
+	buff []*Story
+}
+
+func (c *StoryCursor) Next() (s *Story, err error) {
+	if len(c.buff) == 0 {
+		_, err = c.next(&c.buff)
+		if err != nil {
+			return
```

Comment:
@devoid I prefer to always explicitly return values, i.e. always use `return err` instead of just `err`, so please fix that in the code, it's used like that on multiple places.